api/src/__integration_tests__/LogController.test.ts

```diff
@@ -227,10 +227,10 @@ describe('LogController', () => {
         await externalContactRepository.remove(externalContact1.id);
         await externalContactRepository.remove(externalContact2.id);
 
-        expect(spyOngetOngoingEvents).toBeCalledTimes(1);
-        expect(spyOnIsWithinDistance).toBeCalledTimes(0);
-        expect(spyOnNotifyForEvent).toBeCalledTimes(0);
-        expect(spyOnHasNotificationForEvent).toBeCalledTimes(0);
+        expect(spyOngetOngoingEvents).toHaveBeenCalledTimes(1);
```

Comment:
eslint gave me a warning that we should use the correct function name in favour of the `toBeCalledTimes` alias.